deploy/gen-certs.sh

```diff
@@ -7,15 +7,23 @@ OS="`uname`"
 
 # Generate cert
 openssl genrsa -out webhookCA.key 2048
-openssl req -new -key ./webhookCA.key -subj "/CN=${NAME}.${NAMESPACE}.svc" -out ./webhookCA.csr
-openssl x509 -req -days 365 -in webhookCA.csr -signkey webhookCA.key -out webhook.crt
+openssl req -new -key ./webhookCA.key \
+  -subj "/CN=${NAME}.${NAMESPACE}.svc" \
+  -addext "subjectAltName = DNS:${NAME}.${NAMESPACE}.svc" \
+  -out ./webhookCA.csr
+openssl x509 -req \
+  -extfile <(printf "subjectAltName=DNS:%s.%s.svc" "${NAME}" "${NAMESPACE}") \
+  -days 365 \
+  -in webhookCA.csr \
+  -signkey webhookCA.key \
+  -out webhook.crt
 
 # Generate cert secret
 kubectl -n kubesec create secret generic \
```

Comment:
If the command depends on the target version of Kubernetes, the script should have an option/switch to manage it accordingly.

Reply:
Will test the PR against previous versions of Kubernetes to test that the certificates work with previous versions.

Reply:
The test worked on versions 1.16 - 1.20 of Kubernetes.